src/drift.core/model/TeamProject.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+
+namespace Rangers.Antidrift.Drift.Core
+{
+    public class TeamProject
+    {
+        public string[] AgentPools { get; set; } = new string[0];
```

Comment:
Should this field already be in here ? Shouldn't it be part once we start the pipelines ?

Reply:
You are right, came as part of my thoughts. The property there is not blocking in my opinion though.